nodejs/test/index.spec.js

```diff
@@ -3,6 +3,27 @@ var Message = require('./models/sample_types').Message;
 var serializer = require('../src/index');
 
 describe('Thrift serializer', function () {
+
+	it('should serialize when compression is not specified', function (done) {
+		var message = new Message({
+			text: 'apple',
+			isError: true,
+			count: 0
+		});
+
+		serializer.write(message, serializer.Compression.Disable, function (err, bytes) {
+			expect(err).to.be.null;
+
+			serializer.read(Message, bytes, true, function (err, msg) {
```

Comment:
I know that by using 4 arguments we disable compression, but I'd still write 

```
serializer.read(Message, bytes, Compression.Disable, function (err, msg) {})
```

so people copying the example will use something meaningful.